src/discolight/augmentations/coarsedropout.py

```diff
@@ -0,0 +1,63 @@
+import random
+import math
+import numpy as np
+from discolight.params.params import Params
+from .augmentation.types import ColorAugmentation
+from .decorators.accepts_probs import accepts_probs
+
+
+@accepts_probs
+class CoarseDropout(ColorAugmentation):
+    """
+    Randomly erases a rectangular area in the given image.
+    """
+
+    def __init__(self, p, num):
```

Comment:
Perhaps you could make the parameter names a bit more descriptive:
```suggestion
    def __init__(self, deleted_area, num_rectangles):
```